src/executables/sysrepocfg.c

```diff
@@ -78,8 +78,9 @@ help_print(void)
             "\n"
             "Available options:\n"
             "  -d, --datastore <datastore>  Datastore to be operated on, \"running\" by default (\"running\", \"startup\",\n"
-            "                               \"candidate\", \"operational\", or \"factory-default\"). Accepted by\n"
-            "                               import, export, edit, copy-from op.\n"
+            "                               \"candidate\", \"operational\", or \"factory-default\").\n"
+            "                               or use initials 'r', 's', 'c', 'o', 'f'\n"
+            "                               Accepted by import, export, edit, copy-from op.\n"
```

Comment:
Please keep the correct text flow. There is a leftover period and you continue the sentence, which is not properly finished with a period.

Reply:
```sh
Available options:
  -d, --datastore <datastore>  Datastore to be operated on, "running" by default ("running", "startup",
                               "candidate", "operational", or "factory-default") or use initials ('r', 's',
                               'c', 'o', or 'f') respectively. Accepted by import, export, edit, copy-from op.

```

This is how it looks now, Hope that is good.